drivers/i3c/master/mipi-i3c-hci/core.c

```diff
@@ -755,7 +755,8 @@ static int i3c_hci_attach_i3c_dev(struct i3c_dev_desc *dev)
 			kfree(dev_data);
 			return ret;
 		}
-		mipi_i3c_hci_dat_v1.set_dynamic_addr(hci, ret, dev->info.dyn_addr);
+		mipi_i3c_hci_dat_v1.set_dynamic_addr(hci, ret,
+						     dev->info.dyn_addr ?: dev->info.static_addr);
```

Comment:
Please use full form of Ternary Condition.
(condition)? True : false;

(dev->info.dyn_addr != NULL) ? dev->info.dyn_addr : dev->info.static_addr;

Reply:
Commit ID "83c645e790e13908c55fa9acd5f1eaa867eab4ca_" has this change already. Drop one of the commits

Reply:
or if it is at different place in the same core.c file. Merge them together.